styles/main.css

```diff
@@ -0,0 +1,364 @@
+body {
+    font-family: Arial, Helvetica, sans-serif, sans-serif;
+}
+
+header {
+    grid-area: header;
+    height: 55px;
+    border-bottom: 2px solid rgb(236, 236, 236);
+    display: grid;
+    grid-template-columns: 150px 1fr 250px;
+    padding: 0 10px 0 10px;
+    margin-bottom: 10px;
+    position: fixed;
+    top: -0.2em;
+    width: 100%;
+    z-index: 1;
+    background: white;
+}
+
+.header__logo {
```

Comment:
Why do you use `__` syntax to name the classes? does it have an special meaning?